swap/tests/happy_path_restart_alice_after_xmr_locked.rs

```diff
@@ -12,7 +12,7 @@ async fn given_alice_restarts_after_xmr_is_locked_resume_swap() {
         let bob_swap = tokio::spawn(bob::run(bob_swap));
 
         let alice_swap = ctx.alice_next_swap().await;
-        let alice_swap = tokio::spawn(alice::run_until(alice_swap, is_xmr_locked));
+        let alice_swap = tokio::spawn(alice::run_until(alice_swap, is_xmr_lock_transaction_sent));
```

Comment:
Given that this commit changes an earlier test, we could just move this change into the commit where `AliceState::XmrLockTransactionSent` was introduced? Seems more correct.